CI/travis.set-build-info.sh

```diff
@@ -59,7 +59,5 @@ if [ -n "$GITHUB_REPOSITORY" ]; then
 fi
 
 export VERSION
-# This no longer has a Git SHA1 appended, which it did previously for anything
```

Comment:
How does this relate to the PR?

Reply:
Not related, just the comment didn't add any useful information so I took it out. We can take it out in another PR.